src/components/card/card.pw.tsx

```diff
@@ -0,0 +1,353 @@
+        paddings
+      );
+    });
+  });
+
+  testData.forEach((stringValue) => {
+    test(`should check ${stringValue} as children`, async ({ mount, page }) => {
+      await mount(<Card>{stringValue}</Card>);
+
+      const cardElement = await card(page);
+
+      await expect(cardElement).toHaveText(stringValue);
+    });
+  });
+
+  ([250, 500, 650, 300] as const).forEach((width) => {
+    test(`should check ${width} width is set on component`, async ({
+      mount,
+      page,
+    }) => {
+      await mount(<CardComponent cardWidth={`${width}px`} />);
+
+      const cardElement = await card(page);
+
+      await assertCssValueIsApproximately(cardElement, "width", width);
+    });
+  });
+
+  ([
+    [1, 2, 2, 2],
+    [3, 1, 4, 0],
+    [4, 2, 2, 2],
+  ] as const).forEach(
+    ([
+      draggableCardItem,
+      columnName,
+      countOfFirstColumn,
+      countOfSecondColumn,
+    ]) => {
+      test(`drag card item ${draggableCardItem} to column ${columnName}`, async ({
+        mount,
+        page,
+      }) => {
+        await mount(<DraggableExample />);
+
+        await draggableCard(page, draggableCardItem - 1).dragTo(
+          draggableContainer(page, columnName)
+        );
+
+        await expect(
+          draggableContainer(page, columnName).locator(
+            `[data-element="draggable-card-${draggableCardItem - 1}"]`
+          )
+        ).toBeVisible();
+
+        const resultForColumnOne = await draggableContainer(page, 1)
+          .locator('[data-component="card"]')
+          .count();
+        const resultForColumnTwo = await draggableContainer(page, 2)
+          .locator('[data-component="card"]')
+          .count();
+
+        await expect(resultForColumnOne).toEqual(countOfFirstColumn);
+
+        await expect(resultForColumnTwo).toEqual(countOfSecondColumn);
+      });
+    }
+  );
+
+  test("should check that the expected dataRole is set on the component", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<CardComponent data-role="cypress" />);
+
+    const cardElement = await card(page);
+
+    await expect(cardElement).toHaveAttribute("data-role", "cypress");
+  });
+
+  test("should check interactive applies the expected styling", async ({
+    mount,
+    page,
+  }) => {
+    await mount(<CardComponent interactive />);
+
+    const cardElement = await card(page);
+    await cardElement.hover();
+
+    await expect(cardElement).toHaveCSS("cursor", "pointer");
+    await expect(cardElement).toHaveCSS(
+      "box-shadow",
+      "rgba(0, 20, 30, 0.2) 0px 5px 5px 0px, rgba(0, 20, 30, 0.1) 0px 10px 10px 0px"
+    );
+  });
+
+  test("should allow custom boxShadow and hoverBoxShadow prop values and they apply the expected styling", async ({
+    mount,
+    page,
+  }) => {
+    await mount(
+      <CardComponent
+        boxShadow="boxShadow400"
+        hoverBoxShadow="boxShadow200"
+        interactive
+      />
+    );
+    const cardElement = await card(page);
```

Comment:
**suggestion: (non-blocking)** a space between the mount and the coasts, assertions etc would look better to follow the same standard as the other tests.

Reply:
Same answer as previous too. Will do. This is the kind of thing I figured the linter would do. Especially if we want or do have this as a pattern.